src/cli/commands/object/get.js

```diff
@@ -16,15 +16,32 @@ module.exports = {
   handler (argv) {
     waterfall([
       (cb) => utils.getIPFS(cb),
-      (ipfs, cb) => ipfs.object.get(argv.key, {enc: 'base58'}, cb),
-      (node, cb) => node.toJSON(cb)
-    ], (err, nodeJson) => {
+      (ipfs, cb) => ipfs.object.get(
+        argv.key,
+        { enc: 'base58' },
+        cb)
+    ], (err, node) => {
```

Comment:
please refrain from these cosmetic changes, I will change it back in the next commit any way and it makes it harder to see the relevant changes.